App/Dependencies/ExposureDetection/ImmuniExposureDetectionExecutor.swift

```diff
@@ -136,7 +136,136 @@ class ImmuniExposureDetectionExecutor: ExposureDetectionExecutor {
       resolve(.fullDetection(now(), summary, exposureInfo, firstProcessedChunk, lastProcessedChunk))
     }
   }
+ 
+  func executeEU(
+      exposureDetectionPeriod: TimeInterval,
+      lastExposureDetectionDate: Date?,
+      exposureDetectionConfiguration: Configuration.ExposureDetectionConfiguration,
+      exposureInfoRiskScoreThreshold: Int,
+      userExplanationMessage: String,
+      enManager: ExposureNotificationManager,
+      tekProvider: TemporaryExposureKeyProvider,
+      now: @escaping () -> Date,
+      isUserCovidPositive: Bool,
+      forceRun: Bool,
+      exposureDetectionEU: [(String, Int?, Date?)]
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Dependencies/ExposureDetection/ImmuniExposureDetectionExecutor.swift;
  Line: 151;
-->

- :warning: swiftlint (0.39.2): Tuples should have at most 2 members. (large_tuple)